package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "revai-node-sdk",
-  "version": "1.1.0",
+  "version": "2.0.0",
```

Comment:
You may need to re-run npm install. It looks like the version number increase didn't get added to package-lock.json

Reply:
Good catch